netlogo-gui/src/main/app/App.scala

```diff
@@ -794,10 +803,12 @@ class App extends
   def handle(e: LoadBeginEvent): Unit = {
     val modelName = workspace.modelNameForDisplay
     errorDialogManager.setModelName(modelName)
-    if(AbstractWorkspace.isApp) frame.setTitle(modelTitle)
-    tabManager.codeTabsPanelOption match {
-      case None           =>
-      case Some(thePanel) => thePanel.codeTabContainer.setTitle(modelTitle)
+    if(AbstractWorkspace.isApp) {
```

Comment:
Prefer space after the `if` before the opening paren.  I know this was in the existing code (or I missed it in a prior review), but either way if we're touching this line we should fix it up.